src/hotspot/cpu/s390/sharedRuntime_s390.cpp

```diff
@@ -2778,7 +2778,7 @@ void SharedRuntime::generate_uncommon_trap_blob() {
 #ifdef ASSERT
   assert(Immediate::is_uimm8(Deoptimization::Unpack_LIMIT), "Code not fit for larger immediates");
   assert(Immediate::is_uimm8(Deoptimization::Unpack_uncommon_trap), "Code not fit for larger immediates");
-  const int unpack_kind_byte_offset = Deoptimization::UnrollBlock::unpack_kind_offset_in_bytes()
+  const int unpack_kind_byte_offset = Deoptimization::UnrollBlock::unpack_kind_offset()
 #ifndef VM_LITTLE_ENDIAN
   + 3
 #endif
```

Comment:
This is breaking build for s390x. 

```
/home/amit/jdk/src/hotspot/cpu/s390/sharedRuntime_s390.cpp: In static member function 'static void SharedRuntime::generate_uncommon_trap_blob()':           
/home/amit/jdk/src/hotspot/cpu/s390/sharedRuntime_s390.cpp:2783:3: error: no match for 'operator+' (operand types are 'ByteSize' and 'int')                 
   const int unpack_kind_byte_offset = Deoptimization::UnrollBlock::unpack_kind_offset()                                                                    
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                    
 #ifndef VM_LITTLE_ENDIAN                                                                                                                                   
 ~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                   
   + 3                                                                                                                                                      
   ^~~                                                                                                                                                      
In file included from /home/amit/jdk/src/hotspot/share/utilities/exceptions.hpp:31,                                                                         
                 from /home/amit/jdk/src/hotspot/share/oops/metadata.hpp:28,                                                                                
                 from /home/amit/jdk/src/hotspot/share/oops/oop.hpp:32,                                                                                     
                 from /home/amit/jdk/src/hotspot/share/runtime/handles.hpp:29,
                 from /home/amit/jdk/src/hotspot/share/code/oopRecorder.hpp:28,                                                                            
                 from /home/amit/jdk/src/hotspot/share/asm/codeBuffer.hpp:28,
                 from /home/amit/jdk/src/hotspot/share/asm/assembler.hpp:28,
                 from /home/amit/jdk/src/hotspot/share/asm/macroAssembler.hpp:28,                                                                          
                 from /home/amit/jdk/src/hotspot/share/asm/macroAssembler.inline.hpp:28,                                                                   
                 from /home/amit/jdk/src/hotspot/cpu/s390/sharedRuntime_s390.cpp:27:                                                                       
/home/amit/jdk/src/hotspot/share/utilities/sizes.hpp:53:20: note: candidate: 'constexpr ByteSize operator+(ByteSize, ByteSize)'                            
 constexpr ByteSize operator + (ByteSize x, ByteSize y) { return in_ByteSize(in_bytes(x) + in_bytes(y)); }                                                 
                    ^~~~~~~~
/home/amit/jdk/src/hotspot/share/utilities/sizes.hpp:53:20: note:   no known conversion for argument 2 from 'int' to 'ByteSize'                            
gmake[3]: *** [lib/CompileJvm.gmk:147: /home/amit/jdk/build/linux-s390x-server-fastdebug/hotspot/variant-server/libjvm/objs/sharedRuntime_s390.o] Error 1  
gmake[3]: *** Waiting for unfinished jobs....
gmake[2]: *** [make/Main.gmk:252: hotspot-server-libs] Error 2

ERROR: Build failed for target 'images' in configuration 'linux-s390x-server-fastdebug' (exit code 2)                                                      
Stopping javac server
```

Reply:
oh, Got it. 

BTW thanks for considering suggestion & for this change as well 🙂